Maestro/subscriptions.json

```diff
@@ -135,6 +141,26 @@
             "/p:NotifyGitHubUsers=dotnet/coreclr-contrib",
             "/verbosity:Normal"
           ]
+        },
+        // This handler will bring the Latest CoreFX master build into WCF master
+        {
+          "maestroAction": "wcf-general",
+          "maestroDelay": "00:10:00",
+          "ScriptFileName": "run.cmd",
+          "Arguments": [
+            "build",
+            "-Project='tests\\build.proj'",
```

Comment:
This line isn't needed for wcf--it looks like you used the coreclr one as an example, but corefx and coreclr are actually different.